### PR TITLE
Add guide to run Piped on Google Cloud VM

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-google-cloud-vm.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    --container-image="gcr.io/pipecd/piped:{{< blocks/latest_version >}}" \
+    --container-arg="piped" \
+    --container-arg="--config-gcp-secret=projects/{GCP_PROJECT_NUMBER}/secrets/vm-piped-config/versions/{SECRET_VERSION}" \
+    --network="{VPC_NETWORK}" \
+    --subnet="{VPC_SUBNET}" \
+    --scopes="https://www.googleapis.com/auth/cloud-platform" \
+    --service-account="vm-piped@{GCP_PROJECT_ID}.iam.gserviceaccount.com"
+  ```
+
+After that, you can see on PipeCD web at `Settings` page that Piped is connecting to control-plane.
```
Comment (khanhtc1202) [inline docs/content/en/docs/operator-manual/piped/installation/installing-on-google-cloud-vm.md:97]: Not sure do we have the ability to watch the connecting status of piped on Setting page currently 👀

Comment (nghialv) [reply docs/content/en/docs/operator-manual/piped/installation/installing-on-google-cloud-vm.md:98]: Yes, we can see when it was started.

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-google-cloud-vm.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    #     publicKeyData: {BASE64_ENCODED_PUBLIC_KEY}
+  ```
+
+- Creating a new secret in [SecretManager](https://cloud.google.com/secret-manager/docs/creating-and-accessing-secrets) to store above configuration data securely
+
+  ``` console
+  gcloud secrets create vm-piped-config --data-file={PATH_TO_CONFIG_FILE}
+  ```
+
+- Creating a new Service Account for Piped and giving it the ability to access above secret
```
Comment (khanhtc1202) [inline docs/content/en/docs/operator-manual/piped/installation/installing-on-google-cloud-vm.md:72]:
```suggestion
- Creating a new Service Account for Piped and giving it the ability to access the above secret
```

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-google-cloud-vm.md (hunk 1 above)
Comment (khanhtc1202) [reply docs/content/en/docs/operator-manual/piped/installation/installing-on-google-cloud-vm.md:98]: Ah, I remembered that 👍

**Comment by khanhtc1202** [comment]
Nice work
/lgtm

File: docs/content/en/docs/operator-manual/piped/installation/installing-on-google-cloud-vm.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+linkTitle: "Installing on Google Cloud VM"
+weight: 2
+description: >
+  This page describes how to install Piped on Google Cloud VM.
+---
+
+## Prerequisites
+
+##### Having piped's ID and Key strings
+- Ensure that the `piped` has been registered and you are having its PIPED_ID and PIPED_KEY strings.
```
Comment (nakabonne) [inline docs/content/en/docs/operator-manual/piped/installation/installing-on-google-cloud-vm.md:12]: nit
```suggestion
- Ensure that the `piped` has been registered and you are having its `PIPED_ID` and `PIPED_KEY` strings.
```

**Comment by nakabonne** [comment]
LGTM except for just one nit

**Pull request merged**